osbs/build/build_request.py

```diff
@@ -486,7 +486,13 @@ def render_koji_promote(self, use_auth=None):
 
             if use_auth is not None:
                 self.dj.dock_json_set_arg('exit_plugins', 'koji_promote',
-                                          'use_auth', use_auth)
+                                            'use_auth', use_auth)
+
+            if self.spec.koji_use_kerberos.value:
```

Comment:
OK, so I prefer `koji_use_kerberos` and the `koji_*` keytab and principal parameters, and we just need to check that those parameters are set.